tests/snmp/test_snmp_psu.py

```diff
@@ -21,8 +21,8 @@ def test_snmp_numpsu(duthosts, rand_one_dut_hostname, localhost, creds):
 
 
 @pytest.mark.bsl
-def test_snmp_psu_status(duthosts, rand_one_dut_hostname, localhost, creds):
-    duthost = duthosts[rand_one_dut_hostname]
+def test_snmp_psu_status(duthosts, enum_supervisor_dut_hostname, localhost, creds):
+    duthost = duthosts[enum_supervisor_dut_hostname]
```

Comment:
I think this change will make the test case only applicable to supervisor node. It can't be executed for pizza box DUT.

Reply:
@wangxin in case of pizzabox  the fixture `enum_supervisor_dut_hostname` returns dut[0] so it will not skip for pizzabox

Reply:
I see. Thanks!